### PR TITLE
Add StorageClass for Longhorn deployment

### PR DESCRIPTION
**Pull request by shuo-wu**
https://github.com/longhorn/longhorn/issues/772

### CODE REVIEW

File: deploy/install/02-components/05-storageclass.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: longhorn
+  annotations:
+    storageclass.beta.kubernetes.io/is-default-class: "true"
```
Comment (yasker) [inline deploy/install/02-components/05-storageclass.yaml]: Don't add it as default. Not sure what the user would want.

**Comment by yasker** [comment]
What if user already have the storage class? We need to add some documents around this and CSI.

**Comment by yasker** [comment]
Should we remove https://github.com/longhorn/longhorn-manager/blob/master/examples/storageclass.yaml ?

**Comment by shuo-wu** [comment]
No. I think we can retain this one. I think users will try to find the StorageClass example in `example/` first rather than in `deploy/install/02-components`.

**Pull request merged**